Makefile

```diff
@@ -133,6 +133,7 @@ image_build:
 	scripts/pull_dockerfile_images.sh
 	sed 's/^FROM .*assisted-service.*:latest/FROM $(subst /,\/,${SERVICE})/' Dockerfile.assisted-test-infra | \
 	 $(CONTAINER_COMMAND) build --network=host -t $(IMAGE_NAME) -f- .
```

Comment:
A bit nitty, but can you change it to 2*tab instead of tab+space?
At first I wasn't aware it's a continuation of the previous line